src/app/components/Table/TablePagination.tsx

```diff
@@ -3,33 +3,59 @@ import PaginationItem from '@mui/material/PaginationItem'
 import { FC } from 'react'
 import { useTranslation } from 'react-i18next'
 import { Link, To } from 'react-router-dom'
+import NavigateBeforeIcon from '@mui/icons-material/NavigateBefore'
+import NavigateNextIcon from '@mui/icons-material/NavigateNext'
 
 export type TablePaginationProps = {
-  numberOfPages?: number
+  prevNextOnly?: boolean
+  hasMore?: boolean
   selectedPage: number
   linkToPage: (page: number) => To
 }
 
-export const TablePagination: FC<TablePaginationProps> = ({ numberOfPages, selectedPage, linkToPage }) => {
+// API does not return info about total number of items/pages so we hardcode it
+const numberOfPages = 100
+// -2 means not to render current and next page number
+const negatedSiblingCount = -2
+
+export const TablePagination: FC<TablePaginationProps> = ({
+  prevNextOnly,
+  hasMore,
+  selectedPage,
+  linkToPage,
+}) => {
   const { t } = useTranslation()
 
   return (
     <Pagination
-      count={numberOfPages}
+      boundaryCount={prevNextOnly ? 0 : undefined}
+      siblingCount={prevNextOnly ? negatedSiblingCount : undefined}
+      count={prevNextOnly ? (hasMore ? numberOfPages : selectedPage) : numberOfPages}
       page={selectedPage}
       renderItem={item => (
         <PaginationItem
+          hidden={prevNextOnly && ['end-ellipsis', 'start-ellipsis'].includes(item.type)}
           slots={{
             first: () => <>{t('pagination.first')}</>,
             last: () => <>{t('pagination.last')}</>,
+            previous: () => (
+              <>
+                <NavigateBeforeIcon fontSize="small" /> {t('pagination.previous')}
+              </>
+            ),
```

Comment:
Is the label change intentional on all tables?

| Before | After |
| --- | --- |
|  ![oasis-explorer-frontend-19af130 pages dev_emerald_blocks](https://user-images.githubusercontent.com/3758846/217630499-f66eefc8-40f1-42af-961e-7a95335afba1.png)  | ![localhost_1234_emerald_transactions](https://user-images.githubusercontent.com/3758846/217630509-da445b1d-4402-46da-87d8-700a2b0b7b17.png)  |

Reply:
yes it is, I was asking UX if we want to unify next prev labels